src/libs/ga-analytics.js

```diff
@@ -0,0 +1,39 @@
+const ReactGA = require("react-ga");
+
+const TRACKING_ID = "G-DECWT3BZW"
+
+function init() {
+  // Enable debug mode on the local development environment
+  const isDev = !process.env.NODE_ENV || process.env.NODE_ENV === "development"
+  ReactGA.initialize(TRACKING_ID, { debug: isDev })
+}
+
+function sendEvent(payload) {
+  ReactGA.event(payload);
+}
+
+function sendPageview(path) {
+  ReactGA.set({ page: path })
+  ReactGA.pageview(path);
+}
+
+function sendModalview(path) {
+  ReactGA.modalview(path);
+}
+
+function sendTwitterShare(link_display) {
```

Comment:
these 2 below @anwari666 for social tracking.

even for `open` original link already defined by function